server/audit_math/suite.py

```diff
@@ -534,24 +532,27 @@ def try_n(
         # Otherwise use the sample we've seen so far
         hyp_sample["hyp_round"] = {
             winner: min(
-                math.ceil(
-                    ((n2 - hyp_sample_size) * cumulative_sample[winner])
-                    / hyp_sample_size
+                int(
```

Comment:
why `int` vs `math.ceil`?

Reply:
we want our sample to always be conservative, so we want to round votes for the winner down, not up. This was actually an error I had made in my original implementation. I'm using int because we have to do the cast anyways, and `int(math.floor(`is redundant.

Reply:
looks like `int` and `math.floor` are slightly different, fyi. dunno if it matters here though: https://stackoverflow.com/questions/31036098/what-is-the-difference-between-int-and-floor-in-python-3

Reply:
hmm, tbh truncation probably saves us more headaches. So I think it's fine.